packages/console/src/pages/ApplicationDetails/index.module.scss

```diff
@@ -1,16 +1,21 @@
 @use '@/scss/underscore' as _;
 
 .container {
-  > *:not(:first-child) {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  > :not(:first-child) {
     margin-top: _.unit(4);
   }
 }
 
 .container .header {
-  padding: _.unit(8);
+  flex: 0;
```

Comment:
just curious what is this for?

Reply:
this ensures the header will not be stretched and the `body` section will full-fill the remaining content.
This is temporary and will be refactored once the issue about scroll is done. (LOG-1894)

Reply:
given the body flex: 1 will do?

Reply:
I tried, It doesn't work.

Reply:
It will be like this if we just set the body `flex:1`
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/10806653/158928704-e67fdabe-1107-4897-89e9-607b61e109d3.png">

Reply:
I see, you don't have a height set for this. Nevermind.